### ISSUE TITLE
English translation missing

### ISSUE THREAD

**Issue by lkiesow** [elan-ev/studip-opencast-plugin#716]
Except for a few random words, the plugin is not translated to English and you get many German texts displayed in the otherwise English user interface of Stud.IP.
